pkg/provision/engine.go

```diff
@@ -262,6 +263,21 @@ func withContract(ctx context.Context, contract substrate.NodeContract) context.
 	return context.WithValue(ctx, contractKey{}, contract)
 }
 
+// IsRentedNode returns true if current node is rented
+func IsRentedNode(ctx context.Context) bool {
+	v := ctx.Value(rentKey{})
+	if v == nil {
+		return false
+	}
+
+	return v.(bool)
+}
+
+// mark node as rent internall
```

Comment:
What does rent internal mean?

Reply:
I have no idea what i wanted to say, i will update the docs